README.md

```diff
@@ -1,4 +1,4 @@
-# Marathon Plugin for Jenkins [![Build Status](https://jenkins.mesosphere.com/service/jenkins/buildStatus/icon?job=marathon-plugin-publish-master)](https://jenkins.mesosphere.com/service/jenkins/job/marathon-plugin-publish-master/)
+# Marathon Plugin for Jenkins
```

Comment:
The new build icon can be found here: https://jenkins.mesosphere.com/service/jenkins/buildStatus/icon?job=marathon-jenkins-plugin/marathon-plugin-publish-master

Reply:
Thanks, just updated it.